<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sandwitch_new</class>
 <widget class="QWidget" name="sandwitch_new">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>1392</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1045</width>
    <height>1392</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>1442</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/sandwich_page/background_sandwich_new.png);</string>
   </property>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>246</width>
      <height>362</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>246</width>
      <height>362</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/sandwich_page/new/click_off/t_1.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>246</width>
      <height>362</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>246</width>
      <height>362</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/sandwich_page/new/click_off/t_2.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>330</y>
      <width>246</width>
      <height>362</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>246</width>
      <height>362</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/sandwich_page/new/click_off/t_3.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>740</y>
      <width>246</width>
      <height>362</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>246</width>
      <height>362</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/sandwich_page/new/click_off/t_5.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>750</y>
      <width>246</width>
      <height>362</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>246</width>
      <height>362</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/sandwich_page/new/click_off/t_6.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>740</y>
      <width>246</width>
      <height>362</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>246</width>
      <height>362</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/sandwich_page/new/click_off/t_4.png);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>85</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/button/go_back_btn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>1230</y>
      <width>229</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>229</width>
      <height>101</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/button/Forward Arrow.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
